docs/Development/ReleaseNotes.md

```diff
@@ -107,19 +107,55 @@ This is a major update from VIC 4. The VIC 5.0.0 release aims to have nearly ide
 
 ------------------------------
 
-## VIC 4.2.b [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.22307.svg)](http://dx.doi.org/10.5281/zenodo.22307)
+## VIC 4.2.c [![DOI](https://zenodo.org/badge/doi/10.5281/zenodo.35302.svg)](http://dx.doi.org/10.5281/zenodo.35302)
 
-Source code is available here: [![VIC.4.2.b](https://img.shields.io/badge/VIC-4.2.b-blue.svg)](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.b)
+Source code is available here: [![VIC.4.2.c](https://img.shields.io/badge/VIC-4.2.c-blue.svg)](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.c)
 
 **Release date: (January 22, 2015)**
```

Comment:
That is not correct.  You have [this](https://github.com/tbohn/VIC/blob/update_5.0_release_notes_for_4.2.c/docs/Development/ReleaseNotes.md).  You want: 

4.2.a: Dec 21 2014
4.2.b: Jan 22 2015
4.2.c: Dec 12 2015